MapboxDirections/MBLane.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+
+public enum LaneIndicationType: Int, CustomStringConvertible {
+    case Left
```

Comment:
I forgot this is Swift 2.3. Just keep in mind that we'll need to lowercase the first letters when porting to Swift 3.